src/dex-ui/pages/dao/utils.ts

```diff
@@ -0,0 +1,5 @@
+export function getDaoLinksArray(linksString: string): string[] {
```

Comment:
In Future SC team is removing the key support for DAO links, we would be getting the array direct from SC rather a key value string.